charts/producer-app-cleanup-job/templates/_helpers.tpl

```diff
@@ -17,7 +17,7 @@ If release name contains chart name it will be used as a full name.
 {{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
 {{- else -}}
 {{- $name := default .Values.nameOverride -}}
-{{- printf "%s-clean" $name | trunc 63 | trimSuffix "-" -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
```

Comment:
probably the formatstring is not needed anymore and there's a better way to write it, but I am fine with it

Reply:
I copied it from other charts in this repo. You are probably right but I would keep it for now